### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by darxriggs**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1,2 +1,2 @@
 // Build the plugin using https://github.com/jenkins-infra/pipeline-library
-buildPlugin()
+buildPlugin(configurations: buildPlugin.recommendedConfigurations())
```
Comment (pjdarton) [inline Jenkinsfile]: Could you update the description to explain / link-to-an-explanation of what this does and why it's a good idea?

File: src/main/java/io/jenkins/plugins/extratoolinstallers/installers/AnyOfInstaller.java [hunk 2]
```diff
@@ -218,7 +218,7 @@ public FilePath performInstallation(final ToolInstallation tool, final Node node
 
     /**
      * Logs a message to the build log, indicating the point in our proceedings
-     * that the message relates to, using as little text as possile to do it.
+     * that the message relates to, using as little text as possible to do it.
```
Comment (pjdarton) [inline src/main/java/io/jenkins/plugins/extratoolinstallers/installers/AnyOfInstaller.java]: You have an eye for detail - I appreciate that :grin:

**Comment by pjdarton** [comment]
I appreciate folks putting in effort to tidy up the mess left by others (I wrote AnyOfInstaller and the Authenticated... code, but the remainder predates my involvement), but could you provide a human-readable description of what the functional changes are and why they're a good idea?
Typo corrections and ensuring there's EOLs at end of files I understand, and the boxing changes I'll tolerate (personally I prefer explicit boxing/unboxing but I'll accept implicit as long as a NPE from the line won't be ambiguous), but the pom.xml dependencies changes & Jenkinsfile changes aren't so obviously "correct".  I'd like to know why they're a good idea (and I'd like to know if they're the sort of thing that should be done on the other plugins I'm involved with).

Note: I'm only "the maintainer" of this plugin because "I touched it last" - I needed to add the authenticated & try-any-of stuff.  If you wish to have commit access to this repo then you have my blessing - feel free to quote this reply (and/or link to this) in any request to obtain commit acccess to this repo and/or to be a maintainer of this plugin.

**Comment by darxriggs** [comment]
@pjdarton I updated the respective commit comments.

**Comment by darxriggs** [comment]
But please wait with a merge. I have to have a look at the Java 11 compatibility.

**Comment by darxriggs** [comment]
I had to remove the commit containing the Jenkinsfile change because it would test with Java 11 but compilation would fail. I will create another pull request to fix Java 11 compatibility when this one is merged.

**Comment by pjdarton** [comment]
The commit comments will get squashed a bit during the merge - what I wanted was a better description in the PR description.

As for waiting - do you want me to delay merging _this_ PR, or were you just informing me that there will be a subsequent PR that I'll have to wait for?
FYI I'm going to be in a position to hit the "merge" button for the next 30 hours, then I'm going to be unavailable for 2 weeks, so if there's anything you want merged quickly, you'll have to be very quick...

**Comment by darxriggs** [comment]
This PR is ready to be merged. I removed one commit yesterday. Please merge it.

I will create another one regarding Java 11 thereafter.

**Pull request merged**